src/Xamarin.Android.Build.Tasks/Xamarin.Android.Bindings.Core.targets

```diff
@@ -0,0 +1,169 @@
+    <ItemGroup>
+      <EmbeddedResource Include="@(EmbeddedReferenceJar)">
+        <LogicalName>__reference__%(Filename)%(Extension)</LogicalName>
+      </EmbeddedResource>
+    </ItemGroup>
+  </Target>
+
+  <Target Name="ResolveLibraryProjects" DependsOnTargets="_CreateBindingResourceArchive" />
+
+  <Target Name="AddLibraryJarsToBind" DependsOnTargets="ResolveLibraryProjects">
+    <ItemGroup>
+      <InputJar Include="$(IntermediateOutputPath)library_project_jars\*.jar" />
+    </ItemGroup>
+  </Target>
+
+  <Target Name="CleanBindingsOutput">
+    <Delete Files="$(ApiOutputFile)" />
+    <Delete Files="$(ApiOutputFile).adjusted" />
+    <Delete Files="$(ApiOutputFile).class-parse" />
+    <Delete Files="$(ApiOutputFile).fixed" />
+    <Delete Files="$(ApiOutputFile).adjusted.fixed" />
+    <Delete Files="$(_GeneratorStampFile)" />
+    <Delete Files="@(IntermediateAssembly->'$(OutputPath)%(filename).xml')" />
+    <Delete Files="@(IntermediateAssembly->'$(IntermediateOutputPath)%(filename).xml')" />
+    <RemoveDirFixed Directories="$(GeneratedOutputPath)" />
+    <RemoveDirFixed Directories="$(_AndroidStampDirectory)" />
+    <RemoveDirFixed Directories="$(IntermediateOutputPath)docs" />
+    <RemoveDirFixed Directories="$(IntermediateOutputPath)javadocs" />
+    <RemoveDirFixed Directories="$(IntermediateOutputPath)javasources" />
+  </Target>
+  
+  <Target Name="CleanLibraryProjectIntermediaries">
+    <Delete Files="$(IntermediateOutputPath)__AndroidLibraryProjects__.zip" />
+    <Delete Files="$(_AndroidLibraryImportsCache)" />
+    <Delete Files="$(_AndroidLibraryProjectImportsCache)" />
+    <Delete Files="$(_AndroidLibrayProjectAssemblyMapFile)" />
+    <RemoveDirFixed Directories="$(IntermediateOutputPath)library_project_jars" />
+    <RemoveDirFixed Directories="$(IntermediateOutputPath)library_project_annotations" />
+    <RemoveDirFixed Directories="$(IntermediateOutputPath)$(_LibraryProjectImportsDirectoryName)" />
+    <RemoveDirFixed Directories="$(IntermediateOutputPath)__library_projects__" />
+    <RemoveDirFixed Directories="$(_AndroidLibrayProjectIntermediatePath)" />
+  </Target>
+
+  <Target Name="CleanNativeLibraryIntermediaries">
+    <Delete Files="$(IntermediateOutputPath)__AndroidNativeLibraries__.zip" />
+    <RemoveDirFixed Directories="$(IntermediateOutputPath)native_library_imports" />
+    <RemoveDirFixed Directories="$(IntermediateOutputPath)libs" />
+    <RemoveDirFixed Directories="$(IntermediateOutputPath)$(_NativeLibraryImportsDirectoryName)" />
+  </Target>
+
+  <Import Project="$(MSBuildThisFileDirectory)Xamarin.Android.Bindings.Documentation.targets" />
+  <Import Project="$(MSBuildThisFileDirectory)Xamarin.Android.Bindings.ClassParse.targets" Condition=" '$(AndroidClassParser)' == 'class-parse' " />
+  <Import Project="$(MSBuildThisFileDirectory)Xamarin.Android.Bindings.JarToXml.targets"   Condition=" '$(AndroidClassParser)' == 'jar2xml' " />
```

Comment:
This import should instead be in some ".Legacy.targets" file so it's only done on desktop VS, not .NET 5.